### ISSUE TITLE
(Functionality) Connect Metadata Contract to POA consensus

### ISSUE THREAD

**Issue by rstormsf** [oraclesorg/oracles-dapps-voting#120]
Problem: When new ballot is created for mining key, the metadata is not saved to Metadata contract on ballot finalization
Solution: fix it

**Comment by varasev** [comment]
This issue is the same as https://github.com/poanetwork/poa-network-consensus-contracts/issues/48.

**Comment by rstormsf** [comment]
correct it's in there as well because both sides needs to be changed

**Comment by varasev** [comment]
Yeah, sure. My link above is just for binding both of these issues.
